dockstore-webservice/src/main/java/io/dockstore/webservice/DockstoreWebserviceConfiguration.java

```diff
@@ -686,6 +687,19 @@ public void setMaxConcurrentSessions(Integer maxConcurrentSessions) {
         public Integer getMaxConcurrentSessions() {
             return this.maxConcurrentSessions;
         }
+
+        /**
+         * ElasticSearch refresh interval, see
+         * https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/java-rest-high-indices-put-settings.html
+         * I got an error once with 60s, the AWS recommended, so cut it back to 30s
```

Comment:
this comment seems out of date then

Reply:
D'oh, I had the compose_setup default in mind when I wrote my previous comments.

Reply:
Even worse, I'd changed it locally, but forgotten to push.